java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -279,4 +280,38 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Double> zscore(String key, String member);
+
+    /**
+     * Returns the difference between the first sorted set and all the successive sorted sets.
+     *
```

Comment:
consider adding a `@link` to `zdiffWithScores`

Reply:
Done